cypress-io/cypress
# chore: various flaky + a11y improvements

- Closes N/A

### User facing changelog
Fixes some alt tags and aria attributes.

### Additional details

This PR does several things!
1. Adds a single retry to the percy finalize step. We'll see if this helps with the percy flakiness where it times out during `percy-finalize` even though all the other jobs have finished
1. Skips flaky tooltip visual test. The visual test in question was actually redundant, as it lived in an `afterEach` and the test itself already had skipped `percySnapshot` commands.
1. Fixes some alt tags and aria attributes while I was in the neighborhood

### Steps to test
1. CI is green 🟢 
2. When run locally, <HeaderBarContent/> only shows one type of a11y violation in the test runner (color contrast). No a11y violations related to `alt` tags or `aria-hidden`
3. As far as the percy finalize retry is concerned, the change is based on the following hypothesis:
> When we've seen the `yarn percy build:finalize` command hang, the build DOES exist in Percy (as verified by navigating to it on the Percy website). Theoretically, since the build exists and all the jobs succeeded, simply retrying the step has a good chance of going through the second time. It may not work... but it also may work! We'll see. 

In any case, for the purposes of this PR, the step to test is just to confirm that the [percy finalize step succeeded on retry in this build](https://app.circleci.com/pipelines/github/cypress-io/cypress/43283/workflows/9c1a1739-5c26-4502-be62-9c735b1c1939/jobs/1805420) where I forced an error by adding a typo to the first script

### How has the user experience changed?
No visual changes. Modest improvements to user experience for screenreader users via alt/aria-hidden attributes

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
